torch/_dynamo/utils.py

```diff
@@ -94,20 +94,22 @@ def profile_wrapper(*args, **kwargs):
 
 frame_phase_timing = collections.OrderedDict()
 
-curr_frame = 0
-
+TOTAL_FRAME_COUNTER = 0
+torch._logging._internal.register_formatter_prefix_fn(lambda: f"frame {TOTAL_FRAME_COUNTER}: ")
```

Comment:
nit: I think it's better to keep this more compact, like just `{TOTAL_FRAME_COUNTER}: `, since it's going into *every* log message. I think it will be clear from context.

Reply:
Sure :)